### ISSUE TITLE
Missing app/assets/images directory causes "We're sorry, but something went wrong"

### ISSUE THREAD

**Issue by bikramwp** [rails/rails#38222]
### Steps to reproduce

Create a new Rails app with `--skip-keeps` and/or ensure you delete the app/assets/images directory. Add Application index view and route (`root to: 'application#index'`) and start the app.

### Expected behavior
application#index view should render fine.

### Actual behavior

Browser shows the error message "We're sorry, but something went wrong. If you are the application owner check the logs for more information."

A 500 error occurs and the only error information displayed in the logs is:

```
Processing by ApplicationController#index as HTML
   Rendering application/index.html.erb within layouts/application
   Rendered application/index.html.erb within layouts/application (Duration: 0.6ms | Allocations: 200)
Completed 500 Internal Server Error in 12ms (ActiveRecord: 0.0ms | Allocations: 4474)
 ```

### System configuration
**Rails version**: 6.0.2.1

**Ruby version**: 2.6.5

**Comment by marceloperini** [comment]
@rafaelfranca @bikramwp I can work on this issue.

**Comment by kamipo** [comment]
Fixed by #38250.

**Comment by mm580486** [comment]
@kamipo @bikramwp @marceloperini 
i have same issues , how you handle this problem 
On my server I get a 500 error in production. I checked the production.log but I have only one error without information:
```

I, [2020-04-21T16:05:31.764482 #1]  INFO -- : [02340258-b640-43e6-811b-18b76867f8ad] Processing by SessionsController#login as HTML
I, [2020-04-21T16:05:31.766391 #1]  INFO -- : [02340258-b640-43e6-811b-18b76867f8ad]   Rendering sessions/login.html.erb within layouts/application
I, [2020-04-21T16:05:31.768061 #1]  INFO -- : [02340258-b640-43e6-811b-18b76867f8ad]   Rendered sessions/login.html.erb within layouts/application (Duration: 1.5ms | Allocations: 309)
I, [2020-04-21T16:05:31.791413 #1]  INFO -- : [02340258-b640-43e6-811b-18b76867f8ad] Completed 500 Internal Server Error in 27ms (ActiveRecord: 0.0ms | Allocations: 5899)
```
